source/jormungandr/jormungandr/instance.py

```diff
@@ -156,6 +158,19 @@ def __init__(
 
         self.zmq_socket_type = zmq_socket_type
 
+        self.configure_equipment_details_providers(equipment_details_providers, name)
+
+    def configure_equipment_details_providers(self, equipment_details_providers, instance_name):
+        equipment_providers = [provider['key'] for provider in app.config[str('EQUIPMENT_DETAILS_PROVIDERS')]]
```

Comment:
I'd prefer instantiate equipment_providers in jormungandr.__init__ as `bss_provider_manager`

Reply:
you will need a abstract(or not) class for equipment_providers